portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml

```diff
@@ -0,0 +1,43 @@
+---
+page_title: Edit account details | Portafly
+page_title_description: Page title of the subsection for Edit account details
+body_title: Edit account details
+body_title_description: Body title of the page for Edit account details
+
+# Main area
+
+organization_name_label: Organization name
+organization_name_label_description: Label for the field that holds the name of the organization that owns the account.
+organization_name_validation: {{organization_name_validation}} cannot be blank. Please add your {{organization_name_validation}}.
```

Comment:
```suggestion
organization_name_validation: "<i>Organization name</i> cannot be blank. Please add your organization name."
```

Reply:
Let's use:
* Italics when referring to field names > _Organization name_ cannot be blank.
* Bold letters when referring to buttons or actions > Choose an item to enable the **Delete** button.
* Italic bold when referring to user names. > Hello **_John_**, You have 5 new messages.

WDYT @dfennessy @cntlsn